### PR TITLE
[i18n-CN] Translated some files to implified Chinese  #1915

### PR DESCRIPTION
**Pull request by 2404589803**
Related to https://github.com/huggingface/huggingface_hub/issues/1915.

What is this PR about ?

I translated the following files to implified Chinese :

- toctree.yml
- index.md
- installation.md
- quick-start.md
- git_vs_http.md
- repository.md

What should you check ?

- Grammar and orthograph
- If my pull requests follows GitHub/open source contribution good practices (I am a beginner)

### CODE REVIEW

File: README.md
Comment (Wauplin) [inline README.md:25]: Nice to add a `_cn` README! Could you also add the link to other READMEs? ([README_de.md](https://github.com/huggingface/huggingface_hub/blob/main/README_de.md), [README_ko.md](https://github.com/huggingface/huggingface_hub/blob/main/README_ko.md) and [README_hi.md](https://github.com/huggingface/huggingface_hub/blob/main/README_hi.md)). Thanks in advance!

File: docs/source/cn/_redirects.yml
Comment (Wauplin) [inline docs/source/cn/_redirects.yml:11]: This file is used in the English version of the docs to make legacy urls still compatible with the current documentation (when we renamed some files). So for simplified Chinese it is not necessary to have it given it is a brand new documentation.

File: docs/source/cn/安装.md
Comment (Wauplin) [inline docs/source/cn/安装.md:1]: Could you rename this file to its English name please? While the content must be in the target language (simplified Chinese) for the users, it is still better to have the file architecture in English so that the maintainers know which file contains what. Also, the `_toctree.yml` file is expecting English-based files (see `local: index` means "take the `index.md` local file).

This apply to other translated files and directories. Thanks in advance for making the changes!

**Review by Wauplin — COMMENTED** [review]
Hi @2404589803! Thanks for opening this PR! I am very enthusiastic of having this documentation translated to (simplified) Chinese! I have added a few technical comments about how to structure the PR but will not be able to review the content myself :slightly_smiling_face: Do you know people from the Chinese-speaking community that would be interested in reviewing your work?

**Comment by Wauplin** [comment]
Ping @adeenayakup and @xianbaoqian who will be interested in this! :hugs:

**Review by Wauplin — COMMENTED** [review]
Another comment I forgot: you should update the github workflow files so that `cn` docs are built as well. You should add `cn` in the languages parameter [here](https://github.com/huggingface/huggingface_hub/blob/7319ea4b1df8f0d025dec97bccf5255ba4970d45/.github/workflows/build_documentation.yaml#L16) and [here](https://github.com/huggingface/huggingface_hub/blob/7319ea4b1df8f0d025dec97bccf5255ba4970d45/.github/workflows/build_pr_documentation.yaml#L17).

File: docs/source/cn/_toctree.yml
Comment (xianbaoqian) [inline docs/source/cn/_toctree.yml]: I think that we do need to translate the title so that they can show up in Chinese.

File: README.md
Comment (2404589803) [reply README.md:25]: No problem, I will add it when I have some free time🦾. Thanks!@ Wauplin

File: docs/source/cn/安装.md
Comment (2404589803) [reply docs/source/cn/安装.md:1]: No problem, I'll modify it when I have some free time 🦾. Thanks! @Wauplin

File: docs/source/cn/index（索引）.md
Comment (2404589803) [reply docs/source/cn/index（索引）.md:1]: Sorry, this is a mistake in my work, I will correct it as soon as possible，thanks @ xianbaoqian

File: docs/source/cn/installation（安装）.md
Comment (2404589803) [reply docs/source/cn/installation（安装）.md]: Sorry, this is a mistake in my work, I will correct it as soon as possible，thanks @ xianbaoqian

**Review by adeenayakup — COMMENTED** [review]
Hey @2404589803. Your contribution is very much appreciated! 🤗 Here are some suggestions for the translation, don't hesitate to drop me a message if needed.

File: docs/source/cn/索引.md
Comment (2404589803) [reply docs/source/cn/索引.md:7]: Thank you for your suggestion, I will modify it as soon as possible @adeenayakup

File: docs/source/cn/index（索引）.md
Comment (2404589803) [reply docs/source/cn/index（索引）.md:1]: The file name has been changed back to English @xianbaoqian

File: docs/source/cn/_redirects.yml
Comment (2404589803) [reply docs/source/cn/_redirects.yml]: File deleted @Wauplin

File: docs/source/cn/安装.md
Comment (2404589803) [reply docs/source/cn/安装.md]: File name has been renamed to English @Wauplin

File: README.md
Comment (2404589803) [reply README.md:25]: Chinese readme links have been added to readme files in other languages @Wauplin

**Comment by Wauplin** [comment]
Thanks a lot for making all the requested changes so quickly @2404589803! :pray: From a technical point of view, this PR is good and ready to be merged. From a content point of you (the translated parts), I let you iterate with @xianbaoqian and @adeenayakup 's feedback. Let me know once it's done!

**Comment by 2404589803** [comment]
OK, please wait until I finish the iteration before merging the branches, thank you very much ！@Wauplin

**Comment by 2404589803** [comment]
At the same time, I am very grateful to everyone for your constructive suggestions on my work. Thank you.🤗 @Wauplin @xianbaoqian @adeenayakup

**Comment by 2404589803** [comment]
I have corrected all the above changes, and I also changed the link pointing inside the translated file (this link is used to jump between project files), because if there is no change, it will appear after merging the branch problem, although the review did not point this out.@Wauplin

File: docs/source/cn/installation.md
Comment (xianbaoqian) [inline docs/source/cn/installation.md:36]:
```suggestion
现在您可以从[从 PyPi 注册表](https://pypi.org/project/huggingface-hub/)安装 `huggingface_hub`：
```

File: README.md
Comment (xianbaoqian) [inline README.md:25]:
```suggestion
        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_cn.md">中文（简体）</a> 
```

**Review by Wauplin — APPROVED** [review]
So cool! Love the quick feedback loop. Thanks again everyone for working on this piece. From my point of view it's good to go to let's merge it! :rocket:

**Pull request merged**
